client/control/components/RecipeForm.js

```diff
@@ -57,8 +99,14 @@ export class RecipeForm extends React.Component {
       );
     }
 
+    const isCloning = route && route.isCloning;
+
+    const submitButtonCaption = recipeId && !isCloning ? 'Update Recipe' : 'Add New Recipe';
+
     return (
       <form className="recipe-form" onSubmit={handleSubmit}>
+        { this.renderCloningMessage() }
```

Comment:
Ideally, the button would go alongside the 'preview' and 'history' buttons, though I was having trouble displaying that button dynamically up there. I wasn't too concerned with the design at this point, but I can take care of it.